src/api/client.ts

```diff
@@ -1,6 +1,19 @@
 import axios, { AxiosInstance } from "axios"
 import { Lock, LockOperation, CreateLockOperation } from "./"
 
+
+export async function issueApiKey(username: string, password: string): Promise<string> {
+    let response = await axios.post("https://user-api.gluehome.com/v1/api-keys", {
```

Comment:
Can we add the api Url as constant? 

const API_URL = 'https://user-api.gluehome.com';

```
let response = await axios.post(`{API_URL}/v1/api-keys`...
```